src/Vector.hs

```diff
@@ -13,8 +14,8 @@ sub :: Vector -> Vector -> Vector
 scalarMult :: Double -> Vector -> Vector
 s `scalarMult` (Vector x y z) = Vector (x * s) (y * s) (z * s)
 
-dotProduct :: Vector -> Vector -> Double
-(Vector x y z) `dotProduct` (Vector x1 y1 z1) = x * x1 + y * y1 + z * z1
+dot :: Vector -> Vector -> Double
+(Vector x y z) `dot` (Vector x1 y1 z1) = x * x1 + y * y1 + z * z1
```

Comment:
Hah, I'd never twigged that you can define functions using infix notation like this. makes sense